release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -141,11 +141,31 @@ digraph "org/package:0.1.0" {
 
 ### Breaking changes
 
+### Developer tools updates
+
+#### New features
+
+##### OpenAPI Tool
+Added support for generating client resource methods in the client generation command. The preferred client method type can be chosen using the `--client-methods=<remote(default)|resource>` option.
+  - `$ bal openapi -i <OpenAPI contract> --client-methods=resource`
+  - `$ bal openapi -i <OpenAPI contract> --mode client --client-methods=resource`
+
+#### Improvements
+
+##### OpenAPI Tool
+###### OpenAPI to Ballerina Schema Generation
+Added support to validate the values in generated Ballerina data types concerning the given OpenAPI schema data types' validation rules. For the validation use an API provided by the Ballerina `constraint` package.
```

Comment:
Added support for OpenAPI schema constraint properties in client/service generation. With this improvement, the OpenAPI constraints will be applied as `ballerina/constraint` standard library package annotations when generating Ballerina clients and services from the OpenAPI definition.

Reply:
Shall I use `constraint keywords` since the OpenAPI schema represent these as keywords for specific schema type?